css/src/atomics/width.scss

```diff
@@ -0,0 +1,101 @@
+$widths: 100, 150, 200, 250, 300, 350, 400, 450, 500 !default;
+// do we need a maximum for globally applicable widths?
+// i.e. .width-500 on a 414px wide screen would be a problem
+// do we need min and max? They are used sometimes.
+// But we're definitely generating more of every category of atomic than we need.
+
+// widths
+
+@each $width in $widths {
+	.width-#{$width} {
+		width: #{$width}px !important;
+	}
+}
+
+@include tablet {
+	@each $width in $widths {
+		.width-#{$width}-tablet {
+			width: #{$width}px !important;
+		}
+	}
+}
+
+@include desktop {
+	@each $width in $widths {
+		.width-#{$width}-desktop {
+			width: #{$width}px !important;
+		}
+	}
+}
+
+@include widescreen {
+	@each $width in $widths {
+		.width-#{$width}-widescreen {
+			width: #{$width}px !important;
+		}
+	}
+}
+
+// max-widths
```

Comment:
Adding a note related to what we talked about on teams- would be good to know how folks are using max-width and min-width and whether we want folks to continue using those approaches.

I think the documentation in this PR describes what is available but needs examples around how these classes should be used based on the existing usages in docs-ui.